blueoil/networks/classification/resnet.py

```diff
@@ -170,8 +170,14 @@ def loss(self, softmax, labels):
         labels = tf.cast(labels, tf.float32)
 
         if self.is_debug:
-            labels = tf.Print(labels, [tf.shape(labels), tf.argmax(labels, 1)], message="labels:", summarize=200)
-            softmax = tf.Print(softmax, [tf.shape(softmax), tf.argmax(softmax, 1)], message="softmax:", summarize=200)
+            labels = tf.compat.v1.Print(labels,
+                                        [tf.shape(labels), tf.argmax(labels, axis=1)],
+                                        message="labels:",
+                                        summarize=200)
+            softmax = tf.compat.v1.Print(softmax,
+                                         [tf.shape(softmax), tf.argmax(softmax, axis=1)],
+                                         message="softmax:",
+                                         summarize=200)
```

Comment:
What about this? It's more consistent with the other function calls in this file.
```py
labels = tf.compat.v1.Print(
    labels,
    [tf.shape(labels), tf.argmax(labels, axis=1)],
    message="labels:",
    summarize=200
)
```

Reply:
Thank you.  OK. I fixed it for consistency!